dtbase/functions/arima/__init__.py

```diff
@@ -4,7 +4,7 @@
 
 from azure.functions import HttpRequest, HttpResponse
 
-from dtbase.models.arima.run_pipeline import run_pipeline
+from dtbase.models.arima_old.run_pipeline import run_pipeline

```

Comment:
Ahh yes it is. I haven't tested much to do with the azure functions because we're probably going to redesign it. So for now, I will replace this with that it should now be but I won't test it extensively for now